SU2_CFD/src/interfaces/cht/CConjugateHeatInterface.cpp

```diff
@@ -106,7 +106,9 @@ void CConjugateHeatInterface::GetDonor_Variable(CSolver *donor_solution, CGeomet
         case CONDUCTIVITYMODEL::CONSTANT:
```

Comment:
This is in a branch for incompressible flow, but CoolProp is only used for compressible right?
```suggestion
        case CONDUCTIVITYMODEL::CONSTANT:
        case CONDUCTIVITYMODEL::COOLPROP:
```